src/main/deploy/practiceswerve/modules/physicalproperties.json

```diff
@@ -6,8 +6,8 @@
     "angle": 20
   },
   "conversionFactor": {
-    "angle": 19.2,
-    "drive": 0.059108484
+    "angle": 28.125,
+    "drive": 0.040279076
```

Comment:
This seems to conflict with the changes made in #56 (see [here](https://github.com/robototes/Crescendo2024/pull/56/files#diff-52add4cc008e271f6ea2329c782841a0083933b53d4d7b5457680d246d06b7d2)) - which one is correct?